src/learn.jl

```diff
@@ -24,19 +24,45 @@ function LearnLogger(path, run_name; kwargs...)
     return LearnLogger(path, tensorboard_logger, Dict{String,Any}())
 end
 
+"""
+    log_event!(logger, value::AbstractString)
+
+Logs a string event given by `value` to `logger`.
+"""
+log_event!(logger, value)
+
 function log_event!(logger::LearnLogger, value)
     logged = string(now(), " | ", value)
     TensorBoardLogger.log_text(logger.tensorboard_logger, "events", logged)
     return logged
 end
 
+"""
+    log_plot!(logger, field::AbstractString, plot, plot_data)
+
+Log a `plot` to `logger` under field `field`.
+
+* `plot`: the plot itself
+* `plot_data`: an unstructured dictionary of values used in creating `plot`.
+
+See also [`log_line_series!`](@ref).
+"""
+log_plot!(logger, field::AbstractString, plot, plot_data)
```

Comment:
I think this runs the risk of switching loggers and then existing code breaking (since it relies on being able to pass some number of args); ideally, you should be able to switch loggers without any code changes. I agree about `log_plot!` not being a nice function- I think we should deprecate it, pointing towards `log_line_series!` instead, and then delete it in the next breaking release.

Reply:
yeah I agree with this, especially as we build out more plot logging functionalities we will probably want to deprecate

Reply:
Yeah, sounds good to me---`log_line_series!` and/or `log_image!`.